depcheck.config.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * README
+ * This file contains a subset of critical dependency validation rules for the transformer packages.
+ * There are certain dependencies we need to be careful of introducing as we refactor the transformers,
+ * including access to @aws-amplify/* namespaced packages (namely core, printer),
+ * but we also wish to limit imports to `fs`, and a few other particular places.
+ */
+
+// List of v2 transformer directories.
+const GQL_V2_TRANSFORMER_PACKAGES = [
+  'amplify-graphql-auth-transformer',
+  'amplify-graphql-default-value-transformer',
+  'amplify-graphql-function-transformer',
+  'amplify-graphql-http-transformer',
+  'amplify-graphql-index-transformer',
+  'amplify-graphql-maps-to-transformer',
+  'amplify-graphql-model-transformer',
+  'amplify-graphql-predictions-transformer',
+  'amplify-graphql-relational-transformer',
+  'amplify-graphql-searchable-transformer',
+  'amplify-graphql-transformer-core',
+  'amplify-graphql-transformer-interfaces',
+  'graphql-transformer-common',
+  'graphql-mapping-template',
+];
+
+const TRANSFORMER_RESTRICTED_IMPORTS = [
+  'fs',
+  '@aws-amplify/cli-core',
+  '@aws-amplify/printer',
```

Comment:
Is this different from the `amplify-prompts` package?

Reply:
oh yeah, package names are wrong here - should be @aws-amplify/amplify-cli-core, and @aws-amplify/amplify-prompts

Reply:
Updated